src/MSBuild/CommandLineSwitches.cs

```diff
@@ -237,6 +238,7 @@ bool emptyParametersAllowed
 #if DEBUG
             new ParameterlessSwitchInfo(  new string[] { "waitfordebugger", "wfd" },        ParameterlessSwitch.WaitForDebugger,       null , null             ),
 #endif
+            new ParameterlessSwitchInfo(  new string[] { "graphbuild", "gb" },              ParameterlessSwitch.GraphBuild,            null , null             ),
```

Comment:
Completely open to suggestion for the switch name.

Also, I made it a parameterless switch rather than a parameterized one which takes true|false. I feel like it makes more sense, but I'm open to suggestion here as well.

Reply:
I think I lean toward something more like `/isolateProjects`. If we really want to expose the graph concept I'd probably go with just `/graph`.

Reply:
I'm fine with `/isolateProjects`. It does kind of separate the whole "isolated builds" feature, including exported targets and all that jazz, from the project graph feature, which is probably a good thing but may also cause a little confusion since the features are somewhat related.

Specifically, isolated project builds would use the graph for scheduling purposes when building a set of projects. However there's also the scenario where we're doing a build of 1 project and are somehow importing the exported target results from the project references (eg. the QuickBuild scenario, although I imagine it being used to super fast incremental builds in VS or even command-line as well), which doesn't necessarily have anything to do with the project graph stuff.